provider/views.py

```diff
@@ -255,7 +325,36 @@ def error_response(self, request, error, **kwargs):
 
         return self.render_to_response(ctx, **kwargs)
 
+    def get_implicit_response(self, request, client):
+        """
+        Assumes validated and authorized request with a response_type that is
+        associated with an implicit flow.
+        """
+        data = self.get_data(request)
+
+        lookup_kwargs = {
+            "user": request.user,
+            "client": client,
+            "scope": scope.to_int(*data.get('scope', '').split())
+        }
+
+        if constants.SINGLE_ACCESS_TOKEN:
+            token = self.get_access_token(request, **lookup_kwargs)
+        else:
+            token = self.create_access_token(request, **lookup_kwargs)
+
+        response_data = self.access_token_response_data(token, data['response_type'])
+
+        basepath = data.get("redirect_uri")
+        if not basepath:
+            basepath = token.client.redirect_uri
```

Comment:
One other thing regarding the `redirect_url` (perhaps can be addressed in a separate PR), should we enforce that the supplied `redirect_url` is TLS-protected?  Otherwise, the access token will be sent in the clear.  See the recommendation in http://tools.ietf.org/html/rfc6819#section-4.4.2.1.  @jibsheet FYI - somewhat related to the `SESSION_COOKIE_SECURE` issue.